cmd/admin-handlers.go

```diff
@@ -243,7 +245,8 @@ func (a adminAPIHandlers) ServerInfoHandler(w http.ResponseWriter, r *http.Reque
 
 			serverInfoData, err := peer.cmdRunner.ServerInfoData()
 			if err != nil {
-				errorIf(err, "Unable to get server info from %s.", peer.addr)
+				ctx := logger.SetReqInfo(context.Background(), (&logger.ReqInfo{}).AppendTags("peerAddress", peer.addr))
```

Comment:
we should perhaps bring a different API pattern here like 

```
logger.SetContext(context.Background()).
        SetReqInfo((&logger.ReqInfo{}).AppendTags("peerAddress", peer.addr))
```

Reply:
this code looks incorrect @harshavardhana ... I think you had something else in mind.

The current code looks like:
logger.SetVALUE(ctx, value)
logger.GetVALUE(ctx)

Here value for us is `*ReqInfo`